warrior/Framework/Utils/data_Utils.py

```diff
@@ -750,7 +750,12 @@ def convert2type(value, data_type='str'):
     """
     type_funcs = {'str': str, 'int': int, 'float': float}
     convert = type_funcs[data_type]
-    return convert(value)
+    try:
+        cvalue = convert(value)
+    except Exception as exception:
+        print_exception(exception)
```

Comment:
1. It would be better if we add the warning message instead of printing the whole exception.
2. I see couple of exceptions , one is we are raising and the other one is because of returning "None". Do necessary changes. 
3. Multiple return statements inside single method. we can assign it to a variable called "status" and handle that value in all the methods where this method is been called already.